connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -125,6 +132,39 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
         return embeddedComponentManager.getInitialParams(context)
     }
 
+    /**
+     *
+     */
+    suspend fun onPermissionRequest(context: Context, request: PermissionRequest) {
+        // we only care about camera permissions (audio/video)
+        val permissionsRequested = request.resources.filter {
+            it in listOf(PermissionRequest.RESOURCE_AUDIO_CAPTURE, PermissionRequest.RESOURCE_VIDEO_CAPTURE)
+        }.toTypedArray()
+        if (permissionsRequested.isEmpty()) {
+            request.deny() // no supported permissions were requested, so reject the request
+            logger.debug(
+                "($loggerTag) Denying permission - ${request.resources.joinToString()} are not supported"
+            )
+            return
+        }
+
+        if (checkSelfPermission(context, Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED) {
```

Comment:
Make `EmbeddedComponentManager` fully responsible for the native camera permission and change this to
```suggestion
        if (embeddedComponentManager.hasCameraPermission(context)) {
```
otherwise this code risks becoming out of sync with `embeddedComponentManager.requestCameraPermission(context)`

Reply:
Good call.  @lng-stripe Thoughts on having a single call to `EmbeddedComponentManager` and inlining this into `requestCameraPermission()`? ie:

```
internal suspend fun requestCameraPermission(context: Context): Boolean? {
        if (checkSelfPermission(context, Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED) {
            logger.debug("($loggerTag) Skipping permission request - CAMERA permission already granted")
            return true
        }

        // proceed with explicit check...
    }

Reply:
I think it's worth having a `hasCameraPermission()` function _and_ your change above.

Reply:
Like abstract the above `if` statement into a `hasCameraPermission` function, or keep what I have in that comment and also add a separate `hasCameraPermission()` call in the Controller? The latter feels duplicative to me

Reply:
I think arguments could be made for either. What you have now is fine 👍